torch/testing/_internal/optests/generate_tests.py

```diff
@@ -445,16 +441,16 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
                     self.seen_ops_to_errors[qualname] = []
                 self.run_test_util(func, args_c, kwargs_c)
             except Exception as ex:
-                if should_print_repro():
+                if should_print_better_repro():
                     self.seen_ops_to_errors[qualname].append((ex, func, args, kwargs))
                 else:
-                    self.seen_ops_to_errors[qualname].append((ex, None, None, None))
+                    self.seen_ops_to_errors[qualname].append((ex, func, None, None))
```

Comment:
nit: the comment for `seen_ops_to_errors` look out of date?
```
# Maps qualname -> List[exception]
```

Reply:
good catch, fixed in the PR one above this one